Sources/CoreFoundation/include/module.modulemap

```diff
@@ -1,4 +1,4 @@
-framework module _CoreFoundation [extern_c] [system] {
+module CoreFoundation {
```

Comment:
Can we see if we can just use this modulemap instead of the `module.map` above? If that works, we can delete the `module.map` and stop copying the headers into an intermediate build directory, which would be great. It helps with configure times and makes sure that we can continue working once clang drops support for `module.map` files.